document-parsers/langchain4j-document-parser-llamaparse/src/main/java/dev/langchain4j/data/document/parser/llamaparse/LlmaParseApi.java

```diff
@@ -0,0 +1,57 @@
+package dev.langchain4j.data.document.parser.llamaparse;
+
+import okhttp3.MultipartBody;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Multipart;
+import retrofit2.http.POST;
+import retrofit2.http.Part;
+import retrofit2.http.Path;
+
+interface LlmaParseApi {
+
+    enum JOB_STATUS {
+        PENDING("PENDING"),
+        SUCCESS("SUCCESS"),
+        ERROR("ERROR"),
+        CANCELED("CANCELED");
+
+        private String value;
+        JOB_STATUS(String value) {
+            this.value = value;
+        }
+
+        String value() {
+            return this.value;
+        }
+    }
+
+    final String baseUrl = "https://api.cloud.llamaindex.ai/api/parsing/";
+
+    @POST("upload")
+    @Multipart
+    @Headers({"accept: application/json", "accept: application/json"})
+    Call<LlamaParseResponse> upload(@Part MultipartBody.Part part, @Part MultipartBody.Part parsingInstructions);
+
+    @GET("job/{job_id}/result/json")
+    @Headers({"accept: application/json"})
+    Call<ResponseBody> jsonResult(@Path("job_id") String job_id);
```

Comment:
nit: please use camelCase for java variable names, in all other methods in this class as well
```suggestion
    Call<ResponseBody> jsonResult(@Path("job_id") String jobId);
```